src/fides/api/ops/oauth/api/deps.py

```diff
@@ -2,10 +2,10 @@
 
 from sqlalchemy.orm import Session
 
+from fides.api.ops.db.session import get_db_session
+from fides.api.ops.oauth.api.urn_registry import TOKEN, V1_URL_PREFIX
+from fides.api.ops.oauth.schemas.oauth import OAuth2ClientCredentialsBearer
 from fides.core.config import get_config
-from fides.lib.db.session import get_db_session
-from fides.lib.oauth.api.urn_registry import TOKEN, V1_URL_PREFIX
-from fides.lib.oauth.schemas.oauth import OAuth2ClientCredentialsBearer
 
 
 def get_db() -> Generator[Session, None, None]:
```

Comment:
I think this file should be removed in favor of src/fides/api/ops/api/deps.py. Nothing seems to be using the oauth2_scheme defined in this file, instead preferring the definition here: src/fides/api/ops/util/oauth_util.py, and get_db() is similarly defined in src/fides/api/ops/api/deps.py